specs/electra/beacon-chain.md

```diff
@@ -239,8 +382,16 @@ class BeaconState(Container):
     next_withdrawal_validator_index: ValidatorIndex
     # Deep history valid from Capella onwards
     historical_summaries: List[HistoricalSummary, HISTORICAL_ROOTS_LIMIT]
-    # [New in Electra:EIP6110]
-    deposit_receipts_start_index: uint64
+    deposit_receipts_start_index: uint64  # [New in Electra:EIP6110]
+    deposit_balance_to_consume: Gwei  # [New in Electra:EIP7251]
+    exit_balance_to_consume: Gwei  # [New in Electra:EIP7251]
+    earliest_exit_epoch: Epoch  # [New in Electra:EIP7251]
+    consolidation_balance_to_consume: Gwei  # [New in Electra:EIP7251]
+    earliest_consolidation_epoch: Epoch  # [New in Electra:EIP7251]
+    pending_balance_deposits: List[PendingBalanceDeposit, PENDING_BALANCE_DEPOSITS_LIMIT]  # [New in Electra:EIP7251]
+    # [New in Electra:EIP7251]
+    pending_partial_withdrawals: List[PendingPartialWithdrawal, PENDING_PARTIAL_WITHDRAWALS_LIMIT]
```

Comment:
```suggestion
    pending_partial_withdrawals: List[PendingPartialWithdrawal, PENDING_PARTIAL_WITHDRAWALS_LIMIT] # [New in Electra:EIP7251]
```

Reply:
the linter complains as the line becomes too long in the spec